src/main/java/GuessNumberRoute.java

```diff
@@ -0,0 +1,80 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import game.GuessResponseEntity;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.nonNull;
+
+public class GuessNumberRoute implements Route {
+    private static final Logger LOGGER = LogManager.getLogger(GuessNumberRoute.class);
+    private final UseCaseFactory useCaseFactory;
+    private final JSONSerializer serializer;
+
+    public GuessNumberRoute(UseCaseFactory useCaseFactory, JSONSerializer serializer) {
+        this.useCaseFactory = useCaseFactory;
+        this.serializer = serializer;
+    }
+
+    @Override
+    public Object handle(Request request, Response response) throws Exception {
```

Comment:
čia nėra griežta taisyklė, bet metodai turėtų būt iki kokių 5 eilučių ilgio, todėl reiktų parefactorint šitą kodą